platform/logging/LoggingConfig.ts

```diff
@@ -0,0 +1,166 @@
+import { Schema, typeOfSchema } from '../types';
+import { LogLevelId } from './LogLevel';
+import { Appenders, AppenderConfigType } from './appenders/Appenders';
+
+/**
+ * Separator string that used within nested context name (eg. plugins::pid).
+ */
+const CONTEXT_SEPARATOR = '::';
```

Comment:
That's what I'm using now, still not sure what will be the best. As you remember it's difficult to use `.` as a separator because of the fact that config treats properties with `.` in the name as nested properties and tries to expand them. But from another hand is may be not that bad idea to separate `.` and `::` logically.